src/i18n/fr_FR.ts

```diff
@@ -1145,89 +736,86 @@ export default {
     },
     buildingIcons: [
         'fire',
-        'rss',
-        'hospital',
-        'clinic-medical',
         'graduation-cap',
-        'shield-alt',
+        'ambulance'
+        '',
+        'hospital',
         'helicopter',
+        'shield-alt',
+        'rss',
         'graduation-cap',
+        '',
+        '',
+        '',
+        '',
         'helicopter',
-        'building',
+        'rss',
         '',
-        'ship',
-        'ship',
-        'fire',
-        'stethoscope',
         'shield-alt',
-        'clinic-medical',
-        'plane',
+        '',
+        'fire',
         'shield-alt',
+        'ambulance',
     ],
     pois: [

```

Comment:
Not the right name for in example 'Station-service' I will sent the whole list to you.